libflux/src/semantic/mod.rs

```diff
@@ -11,7 +11,7 @@ mod infer;
 pub mod nodes;
 
 mod sub;
-mod types;
+pub mod types;
```

Comment:
This is required. How can we let somebody visit the semantic graph that contains types if types are not public?

Reply:
You're right this should be public